app/controllers/pages_controller.rb

```diff
@@ -39,6 +39,9 @@ def show
   end
 
   def follow_up
+    if !params[:member_id].present? && recognized_member.try(:id).present?
+      return redirect_to follow_up_member_facing_page_path(@page.id, member_id: recognized_member.id)
+    end
```

Comment:
But why would you need to change the URL params? If I understand correctly you only need the member_id to include it as a hidden field on the action form.